bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/motiondetector/MotionDetectorHandler.java

```diff
@@ -43,10 +48,16 @@ protected void initializeServices() throws BoschSHCException {
         super.initializeServices();
 
         this.createService(LatestMotionService::new, this::updateChannels, List.of(CHANNEL_LATEST_MOTION));
+        this.createService(IlluminanceService::new, this::updateChannels, List.of(CHANNEL_ILLUMINANCE), true);
     }
 
     private void updateChannels(LatestMotionServiceState state) {
         DateTimeType date = new DateTimeType(state.latestMotionDetected);
         updateState(CHANNEL_LATEST_MOTION, date);
     }
+
+    private void updateChannels(IlluminanceServiceState state) {
+        DecimalType illuminance = new DecimalType(state.illuminance);
```

Comment:
By curiosity, is it a percentage or not ?
If a percentage, you could have used `PercentType` and add a default pattern including the % in the XML definition.

Reply:
Thanks for asking 🙂 When I first tested the sensor, I thought the values were in the 0-100 range. But now I wanted to be sure and held a very bright light directly in front of the sensor. The resulting value: `874` 😜 So probably the range is 0 - 1000. I adapted the code and the documentation accordingly.

General question, not related to this PR: we have a few devices that report battery levels. We use system channels with type `system.battery-level` for that, but we don't use `PercentType` values but `DecimalType` values. Should this be corrected in another pull request?

Reply:
> General question, not related to this PR: we have a few devices that report battery levels. We use system channels with type system.battery-level for that, but we don't use PercentType values but DecimalType values. Should this be corrected in another pull request?

You can use both I believe.